includes/definitions/discovery/pepwave.yaml

```diff
@@ -0,0 +1,132 @@
+mib: CELLULAR:DEVICE:PEPVPN-SPEEDFUSION:WAN:WLC
+modules:
+  sensors:
+    pre-cache:
+      data:
+        - oid:
+            - devicePowerSourceName
+            - wanName
+            - pepVpnStatusProfileName
+    temperature:
+      data:
+        -
+          oid: deviceTemperatureCelsius
+          num_oid: ".1.3.6.1.4.1.27662.200.1.1.1.4.4.1.{{ $index }}"
+          descr: System Temperature
+          index: "deviceTemperatureCelsius.{{ $index }}"
+          divisor: 1000
+    count:
+      data:
+        -
+          oid: wlcMaxNumAp
+          num_oid: ".1.3.6.1.4.1.27662.4.1.1.3.{{ $index }}"
+          descr: Max. Number of Supported AP Licensed
+          group: WLC
+          index: "wlcMaxNumAp.{{ $index }}"
+        -
+          oid: wlcNumApProfile
+          num_oid: ".1.3.6.1.4.1.27662.4.1.1.4.{{ $index }}"
+          descr: Number of AP Profile Created
+          group: WLC
+          index: "wlcNumApProfile.{{ $index }}"
+        -
+          oid: wlcNumWlanNetwork
+          num_oid: ".1.3.6.1.4.1.27662.4.1.1.5.{{ $index }}"
+          descr: Number of WLAN Network Created
+          group: WLC
+          index: "wlcNumWlanNetwork.{{ $index }}"
+        -
+          oid: wlcNumApReg
+          num_oid: ".1.3.6.1.4.1.27662.4.1.1.6.{{ $index }}"
+          descr: Number of AP Registered
+          group: WLC
+          index: "wlcNumApReg.{{ $index }}"
+        -
+          oid: wlcNumApOnline
+          num_oid: ".1.3.6.1.4.1.27662.4.1.1.7.{{ $index }}"
+          descr: Number of Online AP
+          group: WLC
+          index: "wlcNumApOnline.{{ $index }}"
+        -
+          oid: wlcNumAssocSta
```

Comment:
This should be a wireless sensor "clients"

Reply:
fixed